src/Sylius/Bundle/ApiBundle/Resources/config/api_platform/resources/admin/PromotionCoupon.xml

```diff
@@ -21,8 +21,12 @@
             <operation
                 name="sylius_api_admin_promotion_coupon_get_collection"
```

Comment:
The name should be adjusted to the change:
```suggestion
                name="sylius_api_admin_promotion_promotion_coupon_get_collection"
```
the same applies to other changed endpoints

Reply:
https://github.com/Sylius/Sylius/pull/17011